geoapps/inversion/potential_fields/magnetic_vector/constants.py

```diff
@@ -795,16 +829,20 @@
     "by_uncertainty": {"one_of": "uncertainty channel"},
     "bz_channel": {"one_of": "data channel"},
     "bz_uncertainty": {"one_of": "uncertainty channel"},
+    "fix_aspect_ratio": {"types": [bool]},
+    "colorbar": {"types": [bool]},
```

Comment:
Also should not have to validate those as they should not be part of the ui.json